src/katgpucbf/fgpu/compute.py

```diff
@@ -97,7 +95,7 @@ def __init__(
             self.pfb_fir = pfb.PFBFIRTemplate(
                 context, taps, self.internal_channels, 32, self.unzip_factor, complex_input=True
             )
-            self.ddc = ddc.DDCTemplate(context, narrowband.taps, narrowband.decimation, dig_sample_bits)
```

Comment:
Has this always been the case and I've only noticed it because of this explicit commit?
And/Or is this related to removing the requirement of `subsampling % taps == 0`?

Reply:
I can't see the full context of the question because it's on an outdated commit, and the snippet Github shows me isn't enlightening. If your question is still relevant can you re-post it against the latest code?